migration/iD11/oracle/202404171125_IDEMPIERE-6110.sql

```diff
@@ -0,0 +1,10 @@
+-- IDEMPIERE-6110
+SELECT register_migration_script('202404171125_IDEMPIERE-6110.sql') FROM dual;
+
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+-- Apr 17, 2024, 11:25:53 AM BRT
+INSERT INTO AD_Message (MsgType,MsgText,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,AD_Message_ID,Value,EntityType,AD_Message_UU) VALUES ('E','Products or charges configured as ''Freight Product'' or ''Freight Charge'' cannot be added to this order due to the combination of delivery via rule and freight cost rule',0,0,'Y',TO_TIMESTAMP('2024-04-17 11:25:53','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2024-04-17 11:25:53','YYYY-MM-DD HH24:MI:SS'),100,nextidfunc(9,'N'),'FreightOrderLineNotAllowed','D','204c8bb9-d002-4beb-bbc3-1d1a11d7471d')
```

Comment:
Hi @VitorHenri 

I tested this successfully.

However, there is a minor mismatch between oracle and postgresql message, the postgresql message has an extra dot at the end.  Can you please fix that?

Reply:
Hi @CarlosRuiz-globalqss, sure! Adjusted!